src/app/components/register/register.component.ts

```diff
@@ -47,7 +47,10 @@ export class RegisterComponent {
       },
       error: (err: any) => {
         console.log(err)
-      }
+      },
+      complete: () => {
+        console.log('Registration request completed')
```

Comment:
Find a toast message to show for stuffs like this

Reply:
currently using ngx-toastr to handle my messages